src/backend/libs/efiling-cso-starter/src/test/java/ca/bc/gov/open/jag/efilingcsostarter/csoAccountServiceImpl/UpdateClientTest.java

```diff
@@ -1,44 +1,25 @@
 package ca.bc.gov.open.jag.efilingcsostarter.csoAccountServiceImpl;
 
-import brooks.roleregistry_source_roleregistry_ws_provider.roleregistry.RegisteredRole;
-import brooks.roleregistry_source_roleregistry_ws_provider.roleregistry.RoleRegistry;
-import brooks.roleregistry_source_roleregistry_ws_provider.roleregistry.RoleRegistryPortType;
-import brooks.roleregistry_source_roleregistry_ws_provider.roleregistry.UserRoles;
-import ca.bc.gov.ag.csows.accounts.AccountFacade;
 import ca.bc.gov.ag.csows.accounts.AccountFacadeBean;
-import ca.bc.gov.ag.csows.accounts.ClientProfile;
 import ca.bc.gov.ag.csows.accounts.NestedEjbException_Exception;
-import ca.bc.gov.open.jag.efilingcommons.exceptions.CSOHasMultipleAccountException;
 import ca.bc.gov.open.jag.efilingcommons.exceptions.EfilingAccountServiceException;
 import ca.bc.gov.open.jag.efilingcommons.model.AccountDetails;
 import ca.bc.gov.open.jag.efilingcsostarter.CsoAccountServiceImpl;
-import ca.bc.gov.open.jag.efilingcsostarter.CsoHelpers;
-import ca.bc.gov.open.jag.efilingcsostarter.mappers.AccountDetailsMapper;
 import org.junit.jupiter.api.*;
 import org.mockito.ArgumentMatchers;
 import org.mockito.Mock;
 import org.mockito.Mockito;
 import org.mockito.MockitoAnnotations;
 
 import java.math.BigDecimal;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.UUID;
 
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
 @DisplayName("UpdateClient")
 public class UpdateClientTest {
 
-    private static final UUID USER_GUID_NO_ROLE = UUID.randomUUID();
-    private static final UUID USER_GUID_WITH_FILE_ROLE = UUID.randomUUID();
-    private static final UUID USER_GUID_WITH_NO_CSO = UUID.randomUUID();
-    private static final UUID USER_GUID_WITH_EJB_EXCEPTION = UUID.randomUUID();
-    private static final UUID USER_GUID_WITH_MULTI_PROFILE = UUID.randomUUID();
-    public static final String DOMAIN = "Courts";
-    public static final String APPLICATION = "CSO";
-    public static final String IDENTIFIER_TYPE = "CAP";
     private static final String INTERNAL_CLIENT_NUMBER = "123";
-
+    private static final String FAIL_INTERNAL_CLIENT_NUMBER = "1234";
+    private static final String INTERNAL_CLIENT_NUMBER1 = "NOTNULL";
```

Comment:
can we rename this var to describe what it is for and how it differs from `INTERNAL_CLIENT_NUMBER`?

Reply:
I will update it to just use the existing INTERNAL_CLIENT_NUMBER. It is just there so it is not null and the client id illegal argument is the focus of the test.